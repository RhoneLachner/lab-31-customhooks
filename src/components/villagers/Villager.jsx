import React from 'react';
import PropTypes from 'prop-types';
import styles from '../containers/containers.css';

const Villager = ({ name, image }) => (
  <figure className={styles.listItems}>
    <img src={image} alt={name} className={styles.listItems}/>
    <figcaption>{name}</figcaption>
  </figure>
);

Villager.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Villager;

