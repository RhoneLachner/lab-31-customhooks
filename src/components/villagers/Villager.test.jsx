import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Villager from './Villager';

describe('Villager component', () => {
  afterEach(() => cleanup());
  it('renders Villager', () => {
    const { asFragment } = render(<Villager />);
    expect(asFragment()).toMatchSnapshot();
  });
});
