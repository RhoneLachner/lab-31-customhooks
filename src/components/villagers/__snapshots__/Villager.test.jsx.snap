// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Villager component renders Villager 1`] = `
<DocumentFragment>
  <figure
    class="listItems"
  >
    <img
      class="listItems"
    />
    <figcaption />
  </figure>
</DocumentFragment>
`;
